### ISSUE TITLE
Debug set_best_effort_node_power_limit() API on lassen

### ISSUE THREAD

**Issue by slabasan** [LLNL/variorum#324]
Unit test is failing for lassen CPU-only and GPU-only builds, possibly need to add a sleep, check if we are calling the correct API.

```
IBM systems may encounter a delay when setting power limits on the node.We could not verify if the power cap was set correctly.
The verification check after 100ms failed.
Please verify again with variorum_print_power_limit().
/usr/workspace/variorum/gitlab-runner/brink2/builds/xYGwyxK_/000/gitlab/variorum/variorum/src/tests/variorum/t_variorum_cap_best_effort_node_power_limit.cpp:15: Failure
```

**Comment by tpatki** [comment]
@slabasan I believe we've fixed this issue. Please confirm and close.

**Comment by slabasan** [comment]
Fixed by #435, specifically https://github.com/LLNL/variorum/commit/3f954e64b82e4bc797ebe02e81f6e1b78cf278ca to add an environment variable.
